From f53cbeec3d0d7dc370d83ca4450663f3e4b64e59 Mon Sep 17 00:00:00 2001
From: Christoph Junghans <junghans@lanl.gov>
Date: Sun, 19 Mar 2017 08:24:37 -0600
Subject: [PATCH] Realm::get_bd_sibling_id: fix return for single thread

Fixes #230
---
 runtime/realm/threads.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/runtime/realm/threads.cc b/runtime/realm/threads.cc
index 2e56039..73d90c3 100644
--- a/runtime/realm/threads.cc
+++ b/runtime/realm/threads.cc
@@ -1343,6 +1343,7 @@ namespace Realm {
     f = fopen(str, "r");
     if(!f) {
       std::cout << "can't read '" << str << "' - skipping";
+      return -1;
     }
     int sib_core_id;
     int count = fscanf(f, "%d", &sib_core_id);
